app/assets/stylesheets/helpers/_govuk-emergency-banner.scss

```diff
@@ -62,6 +61,6 @@
   }
 
   &.local-emergency {
-    background-color: $red;
+    background-color: $turquoise;
```

Comment:
I'd either split this out into it's own commit, or squash into to the first one as it doesn't fit the commit description of making the banner more like other GOV.UK components.